docker/heartbeat/heartbeat.yml

```
@@ -44,3 +44,7 @@ heartbeat.autodiscover:
               urls: ["http://${data.host}:${data.port}"]
               schedule: "@every 5s"
               timeout: 1s
+
+http.enabled: true
+http.host: localhost
+http.port: 5066
```

```suggestion
http.port: 5066

```
Super-nit: newline consistency :)